rust/src/arrow/kernels.rs

```diff
@@ -55,6 +60,11 @@ where
     array
         .iter()
         .enumerate()
+        .filter(|(_, value)| match value {
+            // Remove None and NaN values
```

Comment:
Given that this is at critical path for queries, could you add one benchmark to how much overhead this adds?

Reply:
Is the `panic` from line 71?

```rust
(Some(vx), Some(vy)) => vy.partial_cmp(vx).unwrap(),
```

If so, can we just take care the unwrap there?

Reply:
I couldn't figure out how to handle the unwrap within `max_by`, but I can give it another shot.